src/components /NavBar.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Link, NavLink, Router, useNavigate } from "react-router-dom";
+import {
+  BsPerson,
+  BsExclamationDiamond,
+  BsHeart,
+  BsCalendarCheck,
+} from "react-icons/bs";
+import { CgPill } from "react-icons/cg";
+
+const NavBar = () => {
+  return (
+    <div className="flex drop-shadow-xl mt-5 justify-evenly w-screen">
+      <div className="text-dark-pink text-5xl grid justify-center bg-gradient-to-br bg-white rounded-xl w-1/6 p-1 hover:bg-slate-400">
+        <Link to="/profile">
+          <BsPerson />
+          <div className="text-xs">Profile</div>
+        </Link>
+      </div>
+      <div className="text-dark-pink text-5xl grid justify-center bg-gradient-to-br bg-white rounded-xl w-1/6 p-1 hover:bg-slate-400">
+        <Link to="/alert">
+          <BsExclamationDiamond />
+          <div className="text-xs">Alert</div>
+        </Link>
+      </div>
```

Comment:
I think a good pattern to avoid repetition here is this 
```suggestion
      const navbarItems = [{ icon: <BsExclamationDiamond, text: "Alert", to: "/alert" }, ....]
```

Then you can just map over the array and render these repeating items, which all carry the same styiling.